scheduler/project.clj

```diff
@@ -77,6 +75,15 @@
                  [org.eclipse.jetty/jetty-server "9.2.6.v20141205"]
                  [org.eclipse.jetty/jetty-security "9.2.6.v20141205"]
 
+                 ; TODO: Upgrade netty.
+                 ; Bring in netty. Note that this is brought in in a lot of other places, Exclude it whenever finding another one.
+                 ; Need to make sure all imports are importing the same version.
+                 ;[io.netty/netty-handler "4.1.63.Final"]
+                 ; Netty now uses epoll with a native library. We need to tell lein our architecture so it can bring in the
+                 ; right library architecture. Make sure this version matches.
+                 ;[io.netty/netty-transport-native-epoll "4.1.63.Final" :classifier "linux-x86_64"]
+                 ;[io.netty/netty-transport-native-unix-common "4.1.63.Final" :classifier "linux-x86_64"]
```

Comment:
Do we really need to keep all of these comments around?

Reply:
No, but this is a helpful starting point for an io.netty upgrade. In particular, it explains a gotcha, and and solves a few issues. This is a battle I tactically retreated from for this PR, but expect to need to continue.